airflow/providers/databricks/hooks/databricks.py

```diff
@@ -536,7 +540,7 @@ def delete_repo(self, repo_id: str):
         :param repo_id: ID of Databricks Repos
         :return:
         """
-        repos_endpoint = ("DELETE", f"api/2.0/repos/{repo_id}")
+        repos_endpoint = DELETE_REPO_ENDPOINT[1].format(repo_id=repo_id)
```

Comment:
Same here

Reply:
you could utilize a lambda to handle the formatting 

```python
>>> y = lambda t, repo_id: (t[0], t[1].format(repo_id=repo_id))
>>> y(DELETE_REPO_ENDPOINT, repo_id)
('DELETE', 'api/2.0/repos/fizzbuzz')
```

Reply:
Piggybacking off what @frankcash suggested, how does this change look @hussein-awala? 

`    def update_repo(self, repo_id: str, json: dict[str, Any]) -> dict:
        """
        Updates given Databricks Repos.

        :param repo_id: ID of Databricks Repos
        :param json: payload
        :return: metadata from update
        """
        modify_route = lambda t, repo_id: (t[0], t[1].format(repo_id=repo_id))
        modified_update_repo_endpoint = modify_route(UPDATE_REPO_ENDPOINT, repo_id)
        return self._do_api_call(modified_update_repo_endpoint, json)

    def delete_repo(self, repo_id: str):
        """
        Deletes given Databricks Repos.

        :param repo_id: ID of Databricks Repos
        :return:
        """
        modify_route = lambda t, repo_id: (t[0], t[1].format(repo_id=repo_id))
        modified_delete_repo_endpoint = modify_route(DELETE_REPO_ENDPOINT, repo_id)
        self._do_api_call(modified_delete_repo_endpoint)`
        
Wasn't too sure if it would be more helpful to just create a helper function instead to do this. Let me know what you think!